tests/basic/customer_issues_test.go

```diff
@@ -757,3 +757,195 @@ var _ = Describe("{CreateCloudSnapAndDelete}", func() {
 		AfterEachTest(contexts)
 	})
 })
+
+func getVolumeAttachedNodeAndKill(volName *volume.Volume) (*node.Node, error) {
+	appVol, err := Inst().V.InspectVolume(volName.ID)
+	if err != nil {
+		return nil, err
+	}
+	attachedNode := appVol.AttachedOn
+	log.InfoD("Volume [%v] is attached to Node [%v]", volName.Name, attachedNode)
+
+	nodeDetails, err := node.GetNodeByIP(attachedNode)
+	if err != nil {
+		return nil, err
+	}
+
+	killPxExecErr := KillPxExecUsingPid(nodeDetails)
+	if killPxExecErr != nil {
+		return nil, err
+	}
+
+	killPxStorageErr := KillPxStorageUsingPid(nodeDetails)
+	if killPxStorageErr != nil {
+		return nil, err
+	}
+
+	return &nodeDetails, nil
+}
+
+func isPodStuckNotRunning(nameSpace string) (bool, map[string]string, error) {
+	restartDetails := make(map[string]string)
+	isPodRestarting := false
+	pods, err := GetAllPodsInNameSpace(nameSpace)
+	if err != nil {
+		log.InfoD("Get all pods from Namespace returned error")
+		return false, nil, err
+	}
+	for _, eachPod := range pods {
+		if eachPod.Status.Phase != "Running" && eachPod.Status.Phase != "ContainerCreating" && eachPod.Status.Phase != "Terminating" {
+			restartDetails[eachPod.Name] = fmt.Sprintf("%v", eachPod.Status.Phase)
+			log.InfoD("Pod Status of Pod [%v] is [%v]", eachPod.Name, eachPod.Status.Phase)
+		}
+	}
+	if len(restartDetails) > 0 {
+		isPodRestarting = true
+	}
+	return isPodRestarting, restartDetails, nil
+}
+
+var _ = Describe("{ContainerCreateDeviceRemoval}", func() {
+
+	JustBeforeEach(func() {
+		StartTorpedoTest("ContainerCreateDeviceRemoval",
+			"Test app stuck in Container Creation with Device exists",
+			nil, 0)
+	})
+
+	var contexts []*scheduler.Context
+	stepLog := "App Stuck in ContainerCreation State with Device Exists in the backend"
+	It(stepLog, func() {
+
+		var isPodRestarting bool = false
+		var terminateAll bool = false
+
+		Inst().AppList = []string{"vdbench-sv4-svc"}
```

Comment:
can you add more apps or else we can use the logic  I have  used in PR
https://github.com/portworx/torpedo/pull/2020/files